Validation: description-missing fixes. (#237)
MicrosoftDocs/bingmaps-docs: BingMaps/v8-web-control/modules/directions-module/distanceunit-enumeration.md

---
title: "DistanceUnit Enumeration | Microsoft Docs"

> description: Describes DistanceUnit enumeration, which is related to unit distance, and provides a list of relevant options.

Reviewer: ```suggestion
description: Describes the DistanceUnit enumeration, which can be a distance defined in miles or kilometers.
```
